### DISCUSSION TITLE
Send Later masks BCC addressees

### DISCUSSION THREAD

**Discussion by vohralikp** [Extended-Thunder/send-later#373] [General]
When I use Send Later instead of usual Send, the message copied to the Sent folder does not keep the Bcc addressees.
Is there any setting that I need to adjust to have Bcc addressees still visible in the messages in the Sent folder?

**Comment by jph-sendlater** [comment]
I can reproduce this behavior. It also happens when you use "Put in outbox" and then "File > Send Unsent Messages". The message goes to the outbox with all headers intact (including BCC), but the copy that makes it into the "Sent" folder is the version that went out to the "To:" recipients (and the BCC header is therefore stripped).

This is a bug in Thunderbird. I've opened [a bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1730738) with the Thunderbird developers. Keep an eye out for updates there.

**Comment by jph-sendlater** [comment]
This is [fixed in Thunderbird](https://bugzilla.mozilla.org/show_bug.cgi?id=1730738), but it will of course take a little while for it to make it into a release update. Still, that was an extremely quick response from the Thunderbird devs. Thank you for reporting this.
